jplag.frontend.text/src/main/java/de/jplag/text/ParserAdapter.java

```diff
@@ -1,23 +1,36 @@
 package de.jplag.text;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.util.HashMap;
-import java.util.Map;
-
-import antlr.Token;
-
 import de.jplag.AbstractParser;
 import de.jplag.TokenConstants;
 import de.jplag.TokenList;
+import edu.stanford.nlp.ling.CoreLabel;
+import edu.stanford.nlp.pipeline.CoreDocument;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
 
 public class ParserAdapter extends AbstractParser {
 
     private final Map<String, Integer> tokenTypes = new HashMap<>();
+    private final StanfordCoreNLP pipeline;
     private int tokenTypeIndex = 1; // 0 is FILE_END token, SEPARATOR is not used as there are no methods.
```

Comment:
Seems there is an old bug in the text frontend. Please change this value to 2. Even though `SEPARATOR` is not used, as the constants don't have any identity, the value `1` will still be interpreted as it. Thus, we must not use it for custom token constants.